### DISCUSSION TITLE
Cloud Infrastructure Hackdays: Starting Nov 19!

### DISCUSSION THREAD

**Discussion by jules32** [Openscapes/openscapes.cloud#9] [Announcements]
The [NASA Openscapes Mentors](https://nasa-openscapes.github.io/mentors) are kicking off a series of bi-weekly Hackdays beginning Tuesday, Nov 19 from 9:30 AM - 11:30 PM MST focused on cloud infrastructure development. We aim to foster new contributions through small group work aligning around specific topics or features, and are modeling these after [earthaccess hackdays](https://github.com/nsidc/earthaccess/discussions/440). 

These hackdays by design have no agenda, are self-organizing and work-focused. They are less about  "big ideas for discussion". They work best when a few people show up with concrete things to accomplish or work on, and we get quickly into breakout rooms to cowork on these ideas. 

These hackdays are not specific to any particular tech. They are an evolution of several things: a desire for regular cloud hacking across the NASA Earth science data centers (DAACs), the six-weekly sessions Openscapes has been having with 2i2c, and the biweekly coworking sessions with Carl Boettiger & Eli Holmes. They can complement the JupyterHub Teams cross-NASA monthly meetings led by Tasha Snow. Note that this repository is a place we can coordinate hackdays and ideas that don't have a specific repo home. It is within the Openscapes GitHub Organization, where we support cloud infrastructure for NASA and NOAA, in collaboration with 2i2c. Work can/does also occur asynchronously in-between these hackdays, but hackdays are a recurring opportunity to connect in real-time.

Please post in this thread if you are interested in joining, and we will extend a calendar invitation!

Some ongoing work & ideas: 

- JupyterHub environments and docker images (corn, py-rocket, etc)
- Hub access policies and admin 
- "how much does it cost"? documenting and exploring costs by specific user usecase
- identifying, cataloging, and comparing fledging options; and then using that to build a "fledging" section into the Earthdata Cloud Cookbook

Do you have suggestions on topics or areas of work for these upcoming Hackdays?

- What Issues are you interested in contributing to?
- What features should we be prioritizing?

**Comment by itcarroll** [comment]
Invite please, thank you!

**Comment by jhkennedy** [comment]
I'd like an invite! My Tuesday mornings are a mess, but I'll come when I can, and an invite will help me re-jigger my schedule.

**Comment by sjshriver** [comment]
I'd like an invite too. Thank you!

**Comment by eeholmes** [comment]
Invite please!

**Comment by tsnow03** [comment]
I would love an invite (tsnow03@umd.edu)!

**Comment by jnywong** [comment]
Another invite here please! 🙌

**Comment by jules32** [comment]
Invites sent to all of the above today (on your calendar, not your inbox since that sometimes sends a million emails to Outlook folks)! Also added Yuvi, Carl B, and all the NASA Openscapes Mentors.

**Comment by michele-ornl** [comment]
@jules32 Can you please add Michele to the invite

**Reply by jules32** [reply]
Yes! You should be invited as part of the NASA Mentors Google Group, please let me know if youre' not seeing it.
